tests/core/test_expectation_suite.py

```diff
@@ -598,6 +598,14 @@ def test_expectation_save_callback_can_come_from_any_copy_of_a_suite(
             == updated_column_name
         )
 
+    @pytest.mark.unit
+    def test_expectation_suite_name_can_be_updated(self, empty_cloud_context_fluent):
```

Comment:
Do we need the context here?

Reply:
yes - a suite can't be instantiated outside the context of a `DataContext`